src/book/03-guides/03-hardware/01-working-with-robots.mdx

```diff
@@ -1,7 +1,76 @@
 ---
 section: Guides
 chapter: Hardware
-title: Working with Robots
-description: How to safely handle and work with robots.
+title: Servo alignment and replacement
+description: Safely replacing and aligning servos
 slug: /guides/hardware/working-with-robots
 ---
+
+## Aligning an Ankle
+
+An alignment of an ankle may need to be done if the foot and the leg of the robot aren't perpendicular when standing.
+An ankle alignment may also be necessary when the foot has come loose.
+
+To align an ankle correctly perform the following instructions:
+
+1. On the inner part of the robots leg, unscrew the screws holding the plastic casing to the the ankle and knee servo.
+   Remove the plastic casing, as seen in the following image:
+   ![leg cover](./images/ankle_leg/leg_cover.jpg 'leg cover')
+
+2. There should be a silver disk, the following photo is a close up:
+   ![servo close up](./images/ankle_leg/servo_close.jpg 'servo close up')
+   There are three small indents on the silver disk, two on one side and one on the other. The single dot should be pointing towards the ground when the robot is standing and the other two dots towards the head.
+   These three dots form an arrow, this arrow should be pointing downwards when the robot is standing.
+
+3. Manually move the disk on the servo so it is in the correct orientation
+
+4. Place the plastic cover back on and replace all the screws. When replacing the screws, one hole must be left without a screw to allow for the servo to move. The hole left without a screw is the largest hole, which can be seen on the right of the following image:
+   ![screw hole](./images/angle_leg/screw_holes.jpg 'screw hole')
```

Comment:
This picture is misleading

Reply:
@tayloryoung6396 we were hoping to replace these pictures with renders, could you get them for us?